### PR TITLE
Auto versioning

### PR DESCRIPTION
**Pull request by afermg**
I added support for automated versioning alongside documentation:
- Readme on root: Where does the index lives in Zenodo
- Readme on src: How update the etags when releasing a new version.

I also added the hashes and updated the first version from my fork.

Note that the automation will not work until you add a zenodo access key (with name ZENODO_TOKEN) to the repository secrets. Someone belonging to the Broad Institute Imaging Platform Community must add that (I suggest we use one from a common account, as to not expose anyone's Zenodo Token) I do not have the permissions to add the token, so I'll leave it to someone with admin rights in this repository. Let me know if you need help to add the token.

### CODE REVIEW

**Comment by shntnu** [comment]
@afermg ready for review?

**Comment by afermg** [comment]
Aye! Go for it.

**Review by shntnu — COMMENTED** [review]
Move docs

**Review by shntnu — COMMENTED** [review]
docs: clarify further

File: src/readme.org [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#+TITLE: Releasing a new version
```
Comment (shntnu) [inline src/readme.org:1]: We are not using org-mode markdown elsewhere; switch to GFM?

File: src/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+# Updating new versions
+
+The JUMP profiles that are most up-to-date are indicated on `profile_index.csv`. To release a new set you need only manually update the urls to point to the new location, and potentially change their associated names if it is a new kind of dataset.
+
+
+# Update hashes
+
+After updating an url it will not match the ETag (provided by S3). This
```
Comment (shntnu) [inline src/readme.md:11]: Incomplete sentence here

**Review by shntnu — APPROVED** [review]

**Comment by shntnu** [comment]
@afermg I made some changes directly, for efficiency

Please note I have moved the csv file, and the `src` into a new folder called `manifests`.  We will soon add a bunch of other such files, so we might as well prep for that now by moving to that folder.

**Comment by shntnu** [comment]
We should merge #120 after this

**Comment by afermg** [comment]
I updated the location from where the scripts are called in both the src readme and github actions. Will merge now.

**Pull request merged**